doc/api/errors.md

```diff
@@ -575,6 +629,7 @@ found [here][online].
 [domains]: domain.html
 [event emitter-based]: events.html#events_class_eventemitter
 [file descriptors]: https://en.wikipedia.org/wiki/File_descriptor
+[Node.js Error Codes]: #nodejs-error-codes
```

Comment:
I think it will be `_` not `-`. You can confirm this by building the docs locally, with `make docopen`.